database-composite-types/README.md

```diff
@@ -0,0 +1,172 @@
+# Usage of composite type to store the pipelines
+
+[Composite column types](https://docs.sqlalchemy.org/en/20/orm/composites.html):
```

Comment:
The link is to SQLAlchemy 2.0, while we're currently using 1.4 (and if we move our images from Fedora to CentOS Stream 9 we'll stay with 1.4 for quite some time), so I'd use the 1.4 version of the docs. 

```suggestion
[Composite column types](https://docs.sqlalchemy.org/en/14/orm/composites.html):
```

Does that affect your examples?

Reply:
You marked this as resolved without touching the link or commenting, so unresolving :stuck_out_tongue_winking_eye: